intl.emu

```diff
@@ -85,16 +85,17 @@ location: https://github.com/tc39/proposal-decimal/
           1. Let _sign_ be ~negative~.
           1. Set _x_ to 0.
         1. <ins>If _x_ is a Decimal128 object, then</ins>
-          1. <ins>Perform ? RequireInternalSlot(_O_, [[Decimal128Data]]).</ins>
-          1. <ins>Let _d_ be _O_.[[Decimal128Data]].</ins>
+          1. <ins>Perform RequireInternalSlot(_x_, [[Decimal128Data]]).</ins>
+          1. <ins>Let _d_ be _x_.[[Decimal128Data]].</ins>
           1. <ins>Assert: _d_ is not *NaN*<sub>𝔻</sub>.</ins>
           1. <ins>Assert: _d_ is not *+∞*<sub>𝔻</sub>.</ins>
           1. <ins>Assert: _d_ is not *-∞*<sub>𝔻</sub>.</ins>
           1. <ins>Let _v_ be _d_[0].</ins>
           1. <ins>If _v_ is *-0*<sub>𝔻</sub>, then
             1. <ins>Let _sign_ be ~negative~.</ins>
             1. <ins>Set _x_ to 0.</ins>
-          1. <ins>Else if _v_ < 0, then let _sign_ be ~negative~.</ins>
+          1. <ins>Else,</ins>
+            1. <ins>If _v_ < 0, let _sign_ be ~negative~.</ins>
           1. Else, let _sign_ be ~positive~.</ins>
```

Comment:
This step probably now needs to be indented.

Reply:
Ah, good spot. Fixed.